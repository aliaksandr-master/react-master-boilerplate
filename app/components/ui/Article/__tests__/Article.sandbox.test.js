/* eslint-env browser, jest */
/* eslint-disable react/jsx-no-bind */

import React from 'react';
import renderer from 'react-test-renderer';
import { wrapSandbox } from '../../../../vendor/react-sandbox/sandbox-staging';
import sandboxArticle from '../Article.sandbox';
import Test from '../../../test/Test';



describe('rendering', () => {
  wrapSandbox(sandboxArticle).forEach(({ name, template, props }) => {
    it(`should render sandbox "${name}"`, () => {
      const ArticleSandboxComponent = template();

      const component = renderer.create(
        <Test>
          <ArticleSandboxComponent {...props} />
        </Test>
      );

      expect(component.toJSON()).toMatchSnapshot();
    });
  });
});
