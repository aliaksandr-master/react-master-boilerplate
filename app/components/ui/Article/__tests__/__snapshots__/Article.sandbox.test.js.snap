// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering should render sandbox "default" 1`] = `
<div>
  <article
    className="Article Article--themeDefault Article--sizeDefault"
  >
    <h1>
      Lorem Ipsum
    </h1>
    <p>
      Is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s.
    </p>
    <p>
      Is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s.
    </p>
  </article>
</div>
`;
